com.unity.render-pipelines.universal/Shaders/LitForwardPass.hlsl

```diff
@@ -132,6 +145,15 @@ half4 LitPassFragment(Varyings input) : SV_Target
     UNITY_SETUP_INSTANCE_ID(input);
     UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
 
+#if defined(_PARALLAXMAP)
+#if defined (REQUIRES_TANGENT_SPACE_VIEW_DIR_INTERPOLATOR)
```

Comment:
This seems redundant as you always define REQUIRES_TANGENT_SPACE_VIEW_DIR_INTERPOLATOR if PARALLAXMAP is defined.

Reply:
Updated the code, so its no longer a case for now